### PR TITLE
Bug 1948082: Set unavailable message

### PR DESCRIPTION
**Pull request by jan--f**
* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: pkg/client/status_reporter.go [hunk 1]
```diff
@@ -147,16 +147,13 @@ func (r *StatusReporter) SetFailed(statusErr error, reason string) error {
 	time := metav1.Now()
 	// The Reason should be upper case camelCase (PascalCase) according to the API docs.
 	reason = strings.ToPascalCase(reason)
+	unavailableMessage := "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
```
Comment (dgrisonnet) [inline pkg/client/status_reporter.go:150]: nit: we could put this message in a const at the top of the file

Comment (simonpasquier) [reply pkg/client/status_reporter.go:150]: I agree with Damien, otherwise lgtm :)

**Pull request merged**